src/GraphQL/Execution/ExecutionNode.cs

```diff
@@ -59,6 +59,40 @@ public IObjectGraphType GetParentType(ISchema schema)
 
             return null;
         }
+
+        public IEnumerable<string> Path
+        {
+            get
+            {
+                var pathList = new List<string>();
+                var node = this;
+                while (!(node is RootExecutionNode))
+                {
+                    if (node.PathIndex.HasValue)
+                        pathList.Add(GetStringIndex(node.PathIndex.Value));
+                    else
+                        pathList.Add(node.Field.Name);
+                    node = node.Parent;
+                }
+                pathList.Reverse();
+                return pathList;
+            }
+        }
+
+        private static string GetStringIndex(int index) => index switch
+        {
+            0 => "0",
+            1 => "1",
+            2 => "2",
+            3 => "3",
+            4 => "4",
+            5 => "5",
+            6 => "6",
+            7 => "7",
+            8 => "8",
+            9 => "9",
```

Comment:
Let's extend the list to 16 cases as a tribute to the binary system :)

Reply:
haha sure

Reply:
it shouldn't make much difference anymore, unless someone is using this for extensive logging...